lock/lock_test.go

```diff
@@ -0,0 +1,46 @@
+package lock
+
+import (
+	"github.com/gruntwork-io/terratest/modules/random"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/go-commons/logging"
+	"github.com/stretchr/testify/assert"
+)
+
+var options = Options {
+	logging.GetLogger("test"),
+	"test-dynamodb-lock-string-" + random.UniqueId(),
+	"test-dynamodb-lock-table",
```

Comment:
Use `random.UniqueId()` in the table name too. This ensures (a) that the table creation logic is put through its paces and (b) that if two tests run concurrently, one of them doesn't delete the table the other is using.

Reply:
👍  I think this also means then the test will need to delete the DynamoDB table itself to clean up.